tests/session/session-startTransaction_error-002.phpt

```diff
@@ -0,0 +1,53 @@
+--TEST--
+MongoDB\Driver\Session::startTransaction() with wrong values in options array
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php skip_if_server_version('<', '4.0'); ?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+
+$manager = new MongoDB\Driver\Manager(URI);
+$session = $manager->startSession();
+
+$options = [
+    [ 'readConcern' => 42 ], 
+    [ 'readConcern' => new stdClass ],
+    [ 'readConcern' => new \MongoDB\Driver\WriteConcern( 2 ) ],
+    [ 'readPreference' => 42 ], 
+    [ 'readPreference' => new stdClass ],
+    [ 'readPreference' => new \MongoDB\Driver\ReadConcern( \MongoDB\Driver\ReadConcern::LOCAL ) ],
+    [ 'writeConcern' => 42 ], 
+    [ 'writeConcern' => new stdClass ],
+    [ 'writeConcern' => new \MongoDB\Driver\ReadPreference( \MongoDB\Driver\ReadPreference::RP_SECONDARY ) ],
```

Comment:
I'd suggest two or three new cases:

 * Valid ReadConcern and invalid ReadPreference
 * Valid ReadConcern and invalid WriteConcern (optional)	
 * Valid ReadPreference and invalid WriteConcern	

This would test the memory leak I noted above.

Reply:
Done